docs/en/wallet-attestation.rst

```diff
@@ -68,6 +74,10 @@ Wallet Instance Initialization and Registration
 
     **Federation Check:** The Wallet Instance needs to check if the Wallet Provider is part of the Federation, obtaining its protocol specific Metadata. A non-normative example of a response from the endpoint **.well-known/openid-federation** with the **Entity Configuration** and the **Metadata** of the Wallet Provider is represented within the section `Wallet Provider metadata`_.
 
+.. note::
+
+  The Trust Framework defines that the Wallet Providers though their Trust Chain Root Authorities are anchored in Trust List managed by appointed Supervisory Body or  by a delegated authority. This ensures the integrity and authenticity of wallet solutions are rigorously maintained. These layers of security and oversight create a trusted environment, allowing users to rely on the legitimacy and safety of their wallet instances. The Framework helps prevent fraudulent redirections, protecting user transactions and data.
```

Comment:
part of this must be moved in the section about the trust infrastructure, where we must bring more details about the integrations with superior trust lists and the implementaiton of the national ones

only the part strictly related to the wallert solution can be kept here